doc/developers/bug_triaging.rst

```diff
@@ -64,6 +64,10 @@ can do the following important tasks:
   or needs help (this is typically very important in the context
   of sprints, where the risk is to create many unfinished PRs)
 
+- If a stalled PR is taken over by a newer PR, then label the stalled PR as
+  "Superseded", leave a comment on the stalled PR linking to the new PR, and
+  close the stalled PR.
```

Comment:
to leave some flexibility ?

```suggestion
  possibly close the stalled PR.
```

Reply:
We can say "likely" if we want to a nuance between the two.